ext/RMagick/rmimage.c

```diff
@@ -827,8 +827,8 @@ Image_aref(VALUE self, VALUE key_arg)
  * Update or add image attribute "key".
  *
  * Ruby usage:
- *   - @verbatim Image#["key"] = attr @endverbatim
- *   - @verbatim Image#[:key] = attr @endverbatim
+ *   - @verbatim Image#["key"]= attr @endverbatim
+ *   - @verbatim Image#[:key]= attr @endverbatim
```

Comment:
@Watson1978 should these be something like `image[:key] = attr`?

Reply:
Since it describes the method name in other usage.

https://github.com/rmagick/rmagick/blob/b6a448e5deb557ce794f75b2e65b4f4b537f2d4e/ext/RMagick/rmmain.c#L387